addons/mail/static/src/models/welcome_view/welcome_view.js

```diff
@@ -0,0 +1,75 @@
+/** @odoo-module **/
+
+import { attr } from '@mail/model/model_field';
+import { registerNewModel } from '@mail/model/model_core';
+
+function factory(dependencies) {
+    class WelcomeView extends dependencies['mail.model'] {
+
+        //----------------------------------------------------------------------
+        // Static
+        //----------------------------------------------------------------------
```

Comment:
static is not a category, it's just public or private (it could be though, but it's not what we usually do)